project/BuildDependencies/scripts/0_package.target-win10-arm.list

```diff
@@ -17,7 +17,7 @@ lcms2-2.9-win10-arm-v141-20200105.7z
 libaacs-0.9.0-win10-arm-v141-20200203.7z
 libass-0.15.2-win10-arm-v142-20210920.7z
 libbdplus-0.1.2-win10-arm-v141-20200105.7z
-libbluray-1.1.2-win10-arm-v142-20200706.7z
+libbluray-1.3.2-win10-arm64-v142-20220908.7z
```

Comment:
Seems UWP-ARM package is wrong. Should be: 
`libbluray-1.3.2-win10-arm-v142-20220908.7z`


`12:18:54 libkodi.lib(BlurayDirectory.obj) : error LNK2001: unresolved external symbol bd_init [D:\jenkins\workspace\WIN-UWP-ARM\kodi-build.arm-uwp\kodi.vcxproj]`

But seems that correct package is missing in mirrors

Reply:
yeap I'm trying to get this to build for arm, it looks like it was disabled in the kodi-deps repo